packages/common/src/utils.ts

```diff
@@ -123,6 +123,7 @@ function parseGethParams(json: any, mergeForkIdPostMerge: boolean = true) {
     [Hardfork.London]: { name: 'londonBlock' },
     [Hardfork.MergeForkIdTransition]: { name: 'mergeForkBlock', postMerge: mergeForkIdPostMerge },
     [Hardfork.Shanghai]: { name: 'shanghaiTime', postMerge: true, isTimestamp: true },
+    [Hardfork.ShardingForkDev]: { name: 'shardingForkTime', postMerge: true, isTimestamp: true },
```

Comment:
TODO: we might soon have official name : `Deneb`?

Reply:
I think it's likely to be Cancun for EL (maybe Deneb is the CL fork name). but will wait to put it in code until something semi-official is announced.

Reply:
i dunno, there was talk of having the same name for el<>cl coordinated hfs